ansible/roles/ocp4_workload_rhacm/tasks/workload.yml

```diff
@@ -0,0 +1,131 @@
+---
+# Implement your Workload deployment tasks here
+
+- name: Setting up workload for user
+  debug:
+    msg: "Setting up workload for user ocp_username = {{ ocp_username }}"
+
+- set_fact:
```

Comment:
Once again this should not be necessary since it's automatically done by the playbook that calls the workload.

Reply:
Changed.